src/apig_wsgi.py

```diff
@@ -57,6 +57,7 @@ def get_environ(event, context, binary_support):
     environ = {
         "CONTENT_LENGTH": str(len(body)),
         "HTTP": "on",
+        "HTTP_HOST": "",
```

Comment:
From reading the WSGI spec and Werkzeug source, it seems defaulting `SERVER_NAME` to something non-empty would be preferable to defaulting `HTTP_HOST` to the empty string.

Reply:
Need to specifier either a 
- `HTTP_HOST`
or 
- `SERVER_NAME` AND `SERVER_PORT`  (both can be empty strings)

Reply:
https://github.com/pallets/werkzeug/blob/master/src/werkzeug/wsgi.py#L153

Reply:
Right, I think we should be specifying both `SERVER_NAME` and `SERVER_PORT` already as per the WSGI spec.